### PR TITLE
MEAL APP

### PR DESCRIPTION
**Pull request by isaka-lumato**
## MEAL APP

- [ ] we used JavaScript to make websites dynamic and build basic single-page apps.
- [ ] we used ES6 syntax.
- [ ] we used ES6 modules.
- [ ] we used callbacks and promises.
- [ ] we used webpack.
- [ ] we used JavaScript best practices and language style guides in code.
- [ ] we used AAA pattern for unit tests.
- [ ] we used units tests for a JavaScript app.
- [ ] we Sent and received data from an API.
- [ ] we used API documentation.
- [ ] we aked JavaScript code synchronous.
- [ ] Performed a code review for a team member.
[Video link](https://youtu.be/oXVPhJ_nsuUl)
[Live demo](https://safafa.github.io/Capstone-Javascript/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+  background: linear-gradient(360deg, rgba(0, 128, 49, 0.897), #0e3013ea);
+  height: 90%;
+  overflow-y: auto;
+
+  /* overflow: -moz-scrollbars-none; */
+  padding: 0.5rem 0;
+  font-family: "Kaisei Decol", serif;
+  font-size: 1rem;
+  z-index: 1;
+  border-radius: 5%;
```
Comment (henatan99) [inline src/style.css:160]: - [x] In the popUp window the image does not have the proper size. Kindly change the image height and you can make the overflow property of this `.popUp` class to scroll. 
image
image

File: src/home.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  const card = document.createElement('div');
+  const image = document.createElement('img');
+  const name = document.createElement('h5');
+  const likeB = document.createElement('button');
+  const nameLike = document.createElement('div');
+  const likeSpan = document.createElement('span');
+  const button = document.createElement('button');
+  const reservation = document.createElement('button');
+  card.setAttribute('data-id', idMeal);
+  image.setAttribute('src', strMealThumb);
```
Comment (henatan99) [inline src/home.js:33]: - [x] Please, don't forget to add the `alt` attribute in HTML image elements. It is HTML best practice requirement. :pray:

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @safafa && @isaka-lumato , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/style.css (hunk 1 above)
Comment (safafa) [reply src/style.css:160]: I fixed the image size. The scroll property was already set only the bar was hidden for estetics. You can scroll using the keyboard arrows.

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**